tools/devops/automation/templates/sign-and-notarized/publish-nugets.yml

```diff
@@ -4,6 +4,14 @@ parameters:
 - name: isPR
   type: boolean
 
+- name: repositoryAlias
+  type: string
+  default: self
```

Comment:
If you don't set a default for all parameters except the top one, you'll get an error if you forget to pass the variable on from the caller (instead of scratching your head trying to figure out where you forgot to do it when something fails in mysterious ways).